pandas/tests/apply/test_frame_transform.py

```diff
@@ -162,8 +162,7 @@ def test_transform_method_name(method):
 frame_kernels_raise = [x for x in frame_transform_kernels if x not in wont_fail]
 
 
-# mypy doesn't allow adding lists of different types
-# https://github.com/python/mypy/issues/5492
+@pytest.mark.xfail(strict=False)
```

Comment:
The xfails were because of tests failing due to additional warnings. will remove these and deal with the warnings.